### PR TITLE
Build python 37 artifacts and use them for the docker image

### PR DESCRIPTION
**Pull request by aahmed-se**

### CODE REVIEW

**Comment by aahmed-se** [comment]
run cpp tests

**Comment by aahmed-se** [comment]
run java8 tests

File: docker/pulsar/scripts/install-pulsar-client-37.sh [hunk 1]
```diff
@@ -20,5 +20,5 @@
 
 set -x
 
-WHEEL_FILE=$(ls /pulsar/pulsar-client | grep cp35)
+WHEEL_FILE=$(ls /pulsar/pulsar-client | grep cp37)
 pip3.5 install /pulsar/pulsar-client/${WHEEL_FILE}
```
Comment (merlimat) [inline docker/pulsar/scripts/install-pulsar-client-37.sh]: `pip3.7` or `pip3`

Comment (aahmed-se) [reply docker/pulsar/scripts/install-pulsar-client-37.sh]: fixed

**Comment by aahmed-se** [comment]
run cpp tests

**Comment by aahmed-se** [comment]
run cpp tests

**Comment by aahmed-se** [comment]
run integration tests

**Comment by merlimat** [comment]
run integration tests

**Comment by aahmed-se** [comment]
run integration tests

**Comment by aahmed-se** [comment]
run integration tests

**Comment by aahmed-se** [comment]
run integration tests

**Comment by aahmed-se** [comment]
run cpp tests

**Comment by aahmed-se** [comment]
run cpp tests

**Comment by aahmed-se** [comment]
run cpp tests

**Comment by aahmed-se** [comment]
run integration tests

**Comment by aahmed-se** [comment]
run cpp tests

**Comment by aahmed-se** [comment]
run cpp tests

**Comment by aahmed-se** [comment]
rerun cpp tests

**Comment by aahmed-se** [comment]
run integration tests

**Comment by aahmed-se** [comment]
run integration tests

**Comment by aahmed-se** [comment]
run integration tests

**Pull request merged**